### PR TITLE
Rhel 9 disable built in help system for now

### PR DESCRIPTION
**Pull request by M4rtinK**
This combines a patch from @VladimirSlavik for individual help button hiding with a new config file option that globally disables the built-in help system in Anaconda.

Lastly, use this config file option in the RHEL config file to disable the built-in help system on RHEL 9 for now, due to updated help content not yet being available.

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Review by poncovka — CHANGES_REQUESTED** [review]
The help will be disabled only temporarily. I don't think we need a configuration option for that. The simple workaround for products is to not install the help files or to change the help directory to an empty dir.

**Comment by M4rtinK** [comment]
> The help will be disabled only temporarily. I don't think we need a configuration option for that. The simple workaround for products is to not install the help files or to change the help directory to an empty dir.

This will still not be enough - without help content there will still be the Help! buttons in GUI and IIRC also the "h" shortcuts in the TUI, but they will basically try to short an empty screen due to missing help content and place-holders, which would not be a good UX.

What could be done is to check if help content is available for a given screen when it loads and only show the Help! button/"h" shortcut when there is some content. While this requires some runtime activity per every screen, on the other hand it frankly looks like a good default behavior rather than showing a useless placeholder.

And combined with making sure no help content is installed (that should be doable) we should get the needed behaviour without a config file option.

**Comment by poncovka** [comment]
Still, we don't have a long-term user of this feature. If you really want to use configuration files for this, just set the `help_directory` configuration option to an empty string and use it to check if we can show the help or not. The minimization of the API applies also to the configuration files.

**Comment by M4rtinK** [comment]
> Still, we don't have a long-term user of this feature. If you really want to use configuration files for this, just set the `help_directory` configuration option to an empty string and use it to check if we can show the help or not. The minimization of the API applies also to the configuration files.

Yeah, sounds good, I've updated the PR to check if `help_directory` is set instead of that new option. :)

File: data/product.d/rhel.conf [hunk 1]
```diff
@@ -61,6 +61,12 @@ default_help_pages =
     rhel_help_placeholder.xml
     rhel_help_placeholder.xml
 
+
+# The help system is currently disabled
```
Comment (VladimirSlavik) [inline data/product.d/rhel.conf:65]: Maybe add a FIXME here?

**Review by VladimirSlavik — APPROVED** [review]
LGTM, Thank you!

File: data/product.d/rhel.conf (hunk 1 above)
Comment (M4rtinK) [reply data/product.d/rhel.conf]: Added (and also fixed the the condition to use properly `help_dir` in rhel.conf :P).

**Review by VladimirSlavik — APPROVED** [review]
LGTM again.

**Comment by VladimirSlavik** [comment]
No help = no help buttons feels right. Maybe we could port that to upstream too?

**Comment by M4rtinK** [comment]
> No help files at all = no help buttons feels right. Maybe we could port that to upstream too?
Yeah, I think we should. :)

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Review by jkonecny12 — APPROVED** [review]
Looks good to me!

**Comment by M4rtinK** [comment]
Found one more spoke (Language support) not correctly calling parent `initialize()` but after fixing that all seems fine in both GUI & TUI and in all addon screen. :)

**Pull request merged**